README.md

```diff
@@ -1 +1,40 @@
-# LeaderBoard-project
\ No newline at end of file
+# LeaderBoard-project
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the simple to do list: using webpack that shows of my accomplishments as a software developer.In this project  we learn how to setup the webpack and nodemodules. This project do the to-do-list add , remove and edit function with local storage.
+
+
+
+## Built With
+
+- HTML
+- Css
+- Javascript
+- Es6
+- webpack
+- gitflow
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
# Please we need your ReadMe to look more professional:
- Provide an explanation and links to how to clone this project.
- Provide how to make the project run on your local machine (to set up webpack, also provide the webpack link).
- Provide installation of Packages.Json (npm install etc ...)
-